### DISCUSSION TITLE
Question about selecting subrows

### DISCUSSION THREAD

**Discussion by rdaud** [TanStack/table#3573] [Q&A]
Hi there!

I have used the getSubRows func to change what I want to render as subrows. Now, instead of having an object with the value of "subRows" I'm using "criativos" as the value for rendering subrows. But by changing how subrow works now when I select the parent row, the subrows don't get automatically selected. 

My question is: where do I need to config in order to let automatically select all my subrows again?

Here's an example of my  object data:

{
        veículo: 'Google',
        arquivo: '200x250px, GIF, HTML',
        prazo: '',
        veiculação: '',
        status: 'Em produção',
        criativos: [{
          id: '23432434',
          nome: 'Heineken 2022',
          arquivo: 'url.com/232312',
          prazo: '2021/12/22',
          veiculação: '2021/12/31',
          status: 'Em produção'
        },
        {
          id: '23432434',
          nome: 'Heineken 2023',
          arquivo: 'url.com/232312',
          prazo: '2021/12/22',
          veiculação: '2021/12/31',
          status: 'Em produção'
        }],
        
      },

**Comment by tyler-mitchell** [comment]
You should be able to change the default `subRows` property by doing this: 

```javascript
const instance = useTable(table,
    {
      columns,
      data,
      getSubRows: (original) => { return original.criativos }
    });
```
